integration/integration_test.go

```diff
@@ -2357,6 +2412,9 @@ func runEnvbuilder(t *testing.T, opts runOpts) (string, error) {
 	logChan, errChan := streamContainerLogs(t, cli, ctr.ID)
 	go func() {
 		for log := range logChan {
+			if opts.logbuf != nil {
+				opts.logbuf.WriteString(log)
+			}
```

Comment:
nice 👍  I'll have to go through and use that in other places!

Reply:
I forgot we had another way to get the logs 😂, but alas this seems more convenient.